rllib/policy/torch_policy.py

```diff
@@ -340,7 +368,9 @@ def learn_on_batch(self, postprocessed_batch):
         return {LEARNER_STATS_KEY: grad_info}
 
     @override(Policy)
-    def compute_gradients(self, postprocessed_batch):
+    @DeveloperAPI
+    def compute_gradients(self, postprocessed_batch: SampleBatch) -> Tuple[
```

Comment:
You can use the ModelGradients and ModelWeight.types instead of a generic tensor type

Reply:
+1